docs/quickstart.md

```diff
@@ -92,4 +99,49 @@ for an explanation of the variables (`vars:`) you can put within the example pla
+
+Post installation tasks
+-----------------------
+
+After you have installed Pulp, you might like to also install the [Pulp 3 CLI](https://github.com/pulp/pulp-cli/blob/develop/docs/quickstart.md)
+so that you can use CLI commands.
+
+#### Checking the installation
+
+If the playbook execution went well, it is expected a "*PLAY RECAP*" output with `failed=0` AND `unreachable=0` tasks on each of the hosts from installation. For example:
+```
+PLAY RECAP ********************************************************************************************************************************************
+pulp-host                       : ok=145  changed=1    unreachable=0    failed=0    skipped=93   rescued=0    ignored=0
+```
+
+An example of a **failed** installation would be something like:
+```
+TASK [geerlingguy.postgresql : Ensure PostgreSQL database is initialized.] ********************************************************************************************************************************************
+fatal: [pulp-host]: FAILED! => {"changed": false, "module_stderr": "sudo: unknown user: postgres\nsudo: unable to initialize policy plugin\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
+
+RUNNING HANDLER [geerlingguy.postgresql : restart postgresql] ********************************************************************************************************************************************
+[WARNING]: Ignoring "sleep" as it is not used in "systemd"
+changed: [pulp-host]
+
+PLAY RECAP ********************************************************************************************************************************************
+pulp-host             : ok=22   changed=5    unreachable=0    failed=1    skipped=27   rescued=0    ignored=0
+```
+
+in above example we can see:
+
+ * on "*PLAY RECAP*" that a task failed for host **pulp-host**: `failed=1`
+ * in which task it failed: `TASK [geerlingguy.postgresql : Ensure PostgreSQL database is initialized.`
+ * and why: `fatal: [pulp-host]: FAILED! => {"changed": false, "module_stderr": "sudo: unknown user: postgres\nsudo: unable to initialize policy plugin\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}`
+
+
+#### Checking the installed packages
+
+Depending on the environment, by default, if not defined in the playbook (or at a [custom location](https://docs.pulpproject.org/pulp_installer/customizing/#pulp-installers-types-of-variables))
+the installer will deploy Pulp through a `virtual env` and using `pip`.
+
+To verify the Pulp packages and versions installed by `pulp_installer` run the following commands in the installed Pulp server:
+```
+source /usr/local/lib/pulp/bin/activate
+pip list |grep pulp
+pip show pulp-{container,file,maven,python,rpm} pulpcore
```

Comment:
Do we actually suggest that users do this? I don't feel they typically need to.

They can also check by going to the status page.

Reply:
The idea of this was not to suggest people run it after installation but to provide the steps to understand how to identify where the Pulp packages were installed.
Just as a context, it arose when I saw someone asking:
_"ok, I ran the playbook and after that, I ssh'ed into my pulp host and tried to check the rpm packages, I also ran pip list but could not find any pulp package installed. So, where does pulp_installer install Pulp?"_

Do you think that we should remove this and present the steps to check the status page instead?

Reply:
maybe we can provide both ways and say that the status page is the recommended one